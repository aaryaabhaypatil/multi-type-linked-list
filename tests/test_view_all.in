NEW 3
i
am
programmer
NEW 2
comp2017
no
REMOVE 0
VIEW ALL