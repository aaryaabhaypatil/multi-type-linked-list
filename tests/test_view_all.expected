List 0: i -> am -> programmer
List 1: comp2017 -> no
List 0 has been removed.

Number of lists: 1
List 1
Number of lists: 1
List 1
